pages/apis/graphql/graphql_cookbook.md

```diff
@@ -678,6 +678,23 @@ mutation UpdateSessionDuration {
 }
 ```
 
+### Update when inactive access tokens are revoked
```

Comment:
```suggestion
### Update API access tokens revocation policy
```

@mbelton-buildkite Hello, we are planning to [name this feature](https://github.com/buildkite/buildkite/pull/13085#discussion_r1270106986), feel free to leave suggestion 🙏🏻

Reply:
I'll have a look at that link. I'm 50/50 on that suggestion. While accurate, I think it sounds more complex than it needs to. Ideally, I'd like these headings to match the words someone might use to describe the task they want to do. They shouldn't have to know the feature name. 🤔

Have a think about that and I'm happy for you to update it if you think it helps more.